cmd/admin-helper/listen_windows.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/Microsoft/go-winio"
+)
+
+func listen() (net.Listener, error) {
+	// see https://github.com/moby/moby/blob/46cdcd206c56172b95ba5c77b827a722dab426c5/daemon/listeners/listeners_windows.go
+	// allow Administrators and SYSTEM, plus whatever additional users or groups were specified
+	sddl := "D:P(A;;GA;;;BA)(A;;GA;;;SY)"
+	sid, err := winio.LookupSidByName("crc-users")
+	if err != nil {
+		return nil, err
+	}
+	sddl += fmt.Sprintf("(A;;GRGW;;;%s)", sid)
```

Comment:
if we don't have `crc-users` then this'll fail or it'll continue working?

Reply:
Yes it will fail if the group doesn't exist. 
An error will be shown in the Windows event logs.

Reply:
Okay, and the group will be add as part of setup in https://github.com/code-ready/crc/pull/2008